nroam-utils.el

```diff
@@ -34,6 +34,16 @@
   (let ((inhibit-read-only t))
     (unless (eq ?\n (char-before (1- (point)))) (insert "\n"))))
 
+(defun nroam--insert-heading (level title &optional tags)
+  "Insert a section heading"
```

Comment:
Could you document LEVEL, TITLE and TAGS? (you can use `checkdoc` to help you with docstrings).

Reply:
Oops, done :-)